lib/contentful_model/manageable.rb

```diff
@@ -0,0 +1,104 @@
+module ContentfulModel
+  module Manageable
+    attr_reader :dirty
+
+    def initialize(*args)
+      @dirty = false
+      define_setters
+    end
+
+    def to_management
+      update_management_entry_fields(management_entry)
+    end
+
+    def save
+      result = to_management.save
+      @dirty = false
+      self
+    end
+
+    def publish
+      to_management.publish
+      self
+    end
+
+    protected
+
+    def update_management_entry_fields(management_entry)
+      map_management_fields(management_entry, fields)
+    end
+
+    private
+
+    def management_space
+      @management_space ||= self.class.management(default_locale: locale).spaces.find(space.id)
+    end
+
+    def management_entry
+      # Fetches always in case of Version changes to avoid Validation errors
```

Comment:
Yah, I would also have thought of keeping a list of modified fields. Maybe we should also have a way to get a list of conflicts if people want to implement their own resolution? But I also don't see how to even do that if the management entries come with empty `fields`, I think that would need to be fixed/changed first.

I'm thinking for the time being it would be better to have `save` fail with `VersionMismatch` until we can implement a better approach.